dataactbroker/scripts/createGenerationJobs.py

```diff
@@ -1,47 +0,0 @@
-from dataactcore.interfaces.interfaceHolder import InterfaceHolder
```

Comment:
@alburde1 It's an old data migration from back in the day (before we did data migrations via alembic). So we don't need it going forward, and it will live on in repo history should we ever need to reference it.